multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/Constants.java

```diff
@@ -7,6 +7,7 @@ public class Constants {
     public static final String TASK_ID = "__TASK_ID";
     public static final String INDEXED_STEP_NAME = "__INDEXED_STEP_NAME";
     public static final String VARIABLE_NAME_SERVICE_ID = "__SERVICE_ID";
+    public static final String SUBPROCESS_ID = "subProcessId";
```

Comment:
This constant can be set in Variables for the SUBPROCESS_ID variable.

Reply:
Done